### PR TITLE
Prod release

### PR DESCRIPTION
**Pull request by richtier**

### CODE REVIEW

File: enrolment/views.py [hunk 1]
```diff
@@ -273,6 +274,23 @@ def create_user_profile(self, form):
...
+    def get_referrer_context(self):
+        context = {}
+        referrer_url = self.request.session.get(SESSION_KEY_REFERRER)
+        if referrer_url and referrer_url.startswith(urls.SERVICES_FAB):
+            context = {'fab_referrer': True}
+        self.request.session.pop(SESSION_KEY_REFERRER, None)
+        return context
+
+    def dispatch(self, request, *args, **kwargs):
+        if self.request.session.get(SESSION_KEY_REFERRER) is None:
```
Comment (kowalcj0) [inline enrolment/views.py]: why do we want to introduce `REFERRER_URL` header?
isn't the `HTTP_REFERER` fit for purpose?

File: profile/fab/views.py [hunk 2]
```diff
@@ -198,9 +196,7 @@ def success_url(self):
 class BaseCaseStudyWizardView(NamedUrlSessionWizardView):
     done_step_name = 'finished'
 
-    file_storage = FileSystemStorage(
-        location=os.path.join(settings.MEDIA_ROOT, 'tmp-supplier-media')
-    )
+    file_storage = DefaultStorage()
```
Comment (kowalcj0) [inline profile/fab/views.py]: https://docs.djangoproject.com/en/2.1/ref/files/storage/#django.core.files.storage.DefaultStorage
It seem to me that `DefaultStorage` does exactly the same thing as `FileSystemStorage` but we don't have to define the location. Is that correct @richtier ? if not then what's the reason behind this change?

File: conf/settings.py [hunk 3]
```diff
@@ -332,6 +331,10 @@
     'ENROLMENT_SELECT_BUSINESS_ON': env.bool(
         'FEATURE_ENROLMENT_SELECT_BUSINESS_ENABLED', True
     ),
+    'NEW_HEADER_FOOTER_ON': env.bool(
```
Comment (kowalcj0) [inline conf/settings.py]: these two feature flags aren't used anywhere in this PR. Is that correct?

Comment (richtier) [reply conf/settings.py]: they are used by the new header and footer

File: enrolment/views.py (hunk 1 above)
Comment (richtier) [reply enrolment/views.py]: we're retrieving 

```
            self.request.session[SESSION_KEY_REFERRER] = request.META.get(
                'HTTP_REFERER'
            )
```

and storing it in the session under the `'REFERRER_URL'` key

Comment (richtier) [reply enrolment/views.py]: we're retrieving 

```
            self.request.session[SESSION_KEY_REFERRER] = request.META.get(
                'HTTP_REFERER'
            )
```

and storing it in the session under the `'REFERRER_URL'` key

effectively we're remembering the point of ingress

File: profile/fab/views.py (hunk 2 above)
Comment (richtier) [reply profile/fab/views.py]: I think you're correct.

the intended change was actually to use the file storage provided by white noise.

while this change does not solve the intended problem, it does no harm

File: enrolment/views.py (hunk 1 above)
Comment (richtier) [reply enrolment/views.py]: we're retrieving 

```
            self.request.session[SESSION_KEY_REFERRER] = request.META.get(
                'HTTP_REFERER'
            )
```

and storing it in the session under the `'REFERRER_URL'` key
